src/env/env.c

```diff
@@ -0,0 +1,94 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env.c                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: alopez-g <marvin@42.fr>                    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2022/10/15 18:07:51 by alopez-g          #+#    #+#             */
+/*   Updated: 2022/10/16 01:18:23 by alopez-g         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "env.h"
+
+//easy version
+void	env_unsetvar(char ***env, char *key)
```

Comment:
why do you need char ***env?
can;'t it be char **env?

Reply:
changed! only **env_init**, **env_setvar** and **env_destroy** have char ***, because the can modify the actual char **